documentation/book/con-reconciliation.adoc

```diff
@@ -0,0 +1,6 @@
+[id='reconciliation-{context}']
+= Overview of Reconciliation
+
+Although the operator reacts to all notifications about the desired cluster resources received from the OpenShift or Kubernetes cluster, if the operator is not running, or if a notification is not received for any reason, the desired resources will get out of sync with the state of the running OpenShift or Kubernetes cluster.
```

Comment:
We're going to want to ifdef the references to Kubernetes like we do in the getting started section